tests/test_config_consistency.py

```diff
@@ -178,33 +178,46 @@ def test_query_string_redaction(self):
         assert _get_span_by_tags(trace, expected_tags), f"Span with tags {expected_tags} not found in {trace}"
 
 
-def _get_span(spans, resource_name, tags):
```

Comment:
@zacharycmontoya I replaced `_get_span` with `_get_span_by_tags`. I found that this approach works best with most libraries. Resource names are not set in a consistent way across client libraries. This is something we can work with IDM to address.